### PR TITLE
Adds wdio-phantomjs-service to the docs

### PR DESCRIPTION
**Pull request by cognitom**
## Proposed changes

Adds the document about [wdio-phantomjs-service](https://github.com/cognitom/wdio-phantomjs-service).

## Types of changes

Documentation.

## Further comments

Recently UI libraries need more E2E testing. But Java environment is bit heavy for each testing. I'd like to make such tests easier.

### Reviewers: @christian-bromann

wdio-phantomjs-service is a small helper to run PhantomJS seamlessly when running tests with wdio.

### CODE REVIEW

File: docs/guide/services/phantomjs.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: phantomjs
+category: services
+tags: guide
+index: 2
```
Comment (christian-bromann) [inline docs/guide/services/phantomjs.md]: there is already a service with that index. Please change it to 5

**Comment by christian-bromann** [comment]
🎉  awesome to see people writing services around WebdriverIO.

Minor comment. Also would you add the service also to the [cli interface](https://github.com/webdriverio/webdriverio/blob/master/lib/cli.js#L34-L39) so people who set up their project will be asked about if they want to use your service.

Thanks!

**Comment by christian-bromann** [comment]
Sweet thanks

**Comment by cognitom** [comment]
@christian-bromann thank you for offering about cli interface! I've pushed the updates :smile:

Thank you again for your great product 👍

**Comment by cognitom** [comment]
Oh! Thanks, so quick merge 😆

**Comment by christian-bromann** [comment]
Will bundle a release soon. Guess this weekend.

**Pull request merged**